chipy_org/libs/social_auth_pipelines.py

```diff
@@ -16,6 +16,8 @@ def associate_by_email(*args, **kwargs):
     if email:
         User = get_user_model()
         if User.objects.filter(email=email).exists():
-            msg = ugettext('This email is already in use. First login with your other account and '
-                           'under the top right menu click add account.')
-            raise AuthAlreadyAssociated(backend, msg.format(provider=backend.name))
+            msg = (f"Backend is {backend}. Backend name is {backend.name}."
+                    "This email from is already in use. First login with your "
+                    "other account and under the top right menu click add account.")
```

Comment:
I can't find a __str__ or __rep__ implementation associated with the Backend or and of its inherited classes, so I don't know what it will look like.  I would just remove the `f"Backend is {backend}. Backend name is {backend.name}."` portion of this.  

It would be more helpful if we could say "this email is already associate with the {backend}, please login with that account first and then associate this account with it"

That's a bigger lift and would skip for this pull request. 

I'm approving, but would prefer this change to be made before deploy.  If not, we'll get it in the next one.  

Thanks!

Reply:
I compromised and wrote something a little clearer using just `backend.name`.